stashcp.py

```diff
@@ -557,8 +555,8 @@ def get_best_stashcache():
     else:
         prefix = os.environ.get("OSG_LOCATION", "")
         cache_files = [os.path.join(os.path.dirname(os.path.realpath(__file__)), "caches.json"),
-                       os.path.join(prefix, "/etc/stashcache/caches.json"),
-                       os.path.join(prefix, "/usr/share/stashcache/caches.json")]
+                       os.path.join(prefix, "etc/stashcache/caches.json"),
+                       os.path.join(prefix, "usr/share/stashcache/caches.json")]
```

Comment:
I'm confused by these lines.  Why are you removing the "/" ?

Reply:
Because os.path.join() on Python has terrible semantics:
```
>>> OSG_LOCATION="/opt/osg"
>>> os.path.join(OSG_LOCATION, "etc/stashcache/caches.json")
'/opt/osg/etc/stashcache/caches.json'
>>> os.path.join(OSG_LOCATION, "/etc/stashcache/caches.json")
'/etc/stashcache/caches.json'
```
If an argument to os.path.join starts with a `/` then everything before is thrown away.

Reply:
But I believe we do want it to look in `/etc/...` if prefix is empty.  Though, I agree, the prefix was broken before.

Reply:
Good catch.

Reply:
Better now?